perfmetrics/scripts/testing_on_gke/examples/fio/parse_logs.py

```diff
@@ -91,14 +128,36 @@
   for root, _, files in os.walk(LOCAL_LOGS_LOCATION):
     for file in files:
       per_epoch_output = root + f"/{file}"
+      if not per_epoch_output.endswith(".json"):
+        print(f"ignoring file {per_epoch_output} as it's not a json file")
+        continue
```

Comment:
We are already checking if the per_epoch_output can be loaded as json later in the file, so this test seems redundant.

Reply:
Made it much cleaner in clean-up PR https://github.com/GoogleCloudPlatform/gcsfuse/pull/2369/files .